web/src/styles.css

```diff
@@ -976,6 +970,155 @@ sub {
     text-decoration: underline !important;
 }
 
+.modal-background-overlay{
```

Comment:
There's an `overlay` class that you might be able to re-use (if I remember correctly)